### PR TITLE
Grd 407

### PR DESCRIPTION
**Pull request by xuemeiluo**

### CODE REVIEW

File: commands.txt [hunk 1]
```diff
@@ -0,0 +1,29 @@
+## Commands
+This section lists command(s) run by WORKFLOW workflow
+
+* Running WORKFLOW
+
+=== Description here ===.
```
Comment (pruzanov) [inline commands.txt]: THis is a placeholder, may need to add the actual description here

File: cram.wdl [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+  if (defined(inputCram) == true) {
+    call cramToBam {
+      input:
+        inputCram = inputCram,
+        outputFileNamePrefix = outputFileNamePrefix
+    }
+  }
+
+   output {
+        File? outputBam = cramToBam.outputBam
```
Comment (pruzanov) [inline cram.wdl]: I am not sure if it is Ok to have all outputs optional, perhaps we can output a log file that is not optional here?

File: cram.wdl [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+      url: "https://github.com/samtools/samtools"
+      }
+    ]
+  }
+}
+
+
+task bamToCram {
+  input{
+    File? inputBam
```
Comment (pruzanov) [inline cram.wdl]: Why this is optional?

File: cram.wdl [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+    output_meta: {
+      outputCram: "Converted cram file",
+      outputCramIndex: "Index of the Converted CRAM file"
+    }
+  }
+}
+
+task cramToBam {
+  input{
+    File? inputCram
```
Comment (pruzanov) [inline cram.wdl]: Why this is optional?

File: cram.wdl (hunk 2 above)
Comment (pruzanov) [reply cram.wdl]: File convertionLog = select_first([bamToCram.log, cramToBam.log])

**Review by pruzanov — CHANGES_REQUESTED** [review]

**Review by pruzanov — APPROVED** [review]

**Comment by lheisler** [comment]
lets change the name of this from
cram -> cramConverter

**Review by mlaszloffy — APPROVED** [review]

**Pull request merged**